### PR TITLE
Commander grammar review pass (appx first half)

### PR DESCRIPTION
**Pull request by seandenigris**
I will ask a few questions inline in the text…

### CODE REVIEW

File: Chapters/Commander/Commander.pillar [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 Commander models application actions as first class objects following the Command design pattern.
-Every action is implemented as separate command class (subclass of ==CmdCommand==) with the ==execute== method and all state required for execution.
+Every action is implemented as a separate command class (subclass of ==CmdCommand==) with an ==execute== method and all state required for execution.
 
 !!! An example
 
-Let's look how we can apply commands to control simple contact book.
-The code of example is packaged inside Commander-Examples package. So you can load it and play.
+Let's look at how we can apply commands to control a simple book of contacts.
+The code of example is packaged inside the Commander-Examples package, so you can load it and play along.
```
Comment (seandenigris) [inline Chapters/Commander/Commander.pillar]: I don't see such a package in my local clone

File: Chapters/Commander/Commander.pillar [hunk 2]
```diff
@@ -114,53 +114,53 @@ ContactBookPresenter >> initializeWidgets
 	table items: contactBook contents
 ]]]
 
-Also let's define a reusable book as class side variable
+Also, let's define a reusable book as a class side variable.
```
Comment (seandenigris) [inline Chapters/Commander/Commander.pillar]: It doesn't seem to be on the class side and is not capitalized. Is it supposed to be a class var? Class-side instance var?

File: Chapters/Commander/Commander.pillar [hunk 3]
```diff
@@ -212,29 +212,29 @@ RenameContactCommand >> prepareExecutionInContext: aToolContext
 	newName isEmptyOrNil | (newName = contact name) ifTrue: [ ^ CmdCommandAborted signal ]
 ]]]
 
-By default (with help of Spec integration) Commander will pass presenter instance to aToolContext argument. But in general it is not enough and you will see example later.
+By default (with the help of Spec integration) Commander will pass a presenter instance as the argument (aToolContext). But in general it is not enough and you will see example later.
```
Comment (seandenigris) [inline Chapters/Commander/Commander.pillar]: The last sentence here is jarring. How can we rephrase? Is the point that is covers the simple case, but more advanced scenarios might require more complex solutions?

File: Chapters/Commander/Commander.pillar [hunk 4]
```diff
@@ -94,10 +94,10 @@ command execute.
 
 !!! Adding a simple UI
 
-Before we continue we should write simple UI which will show contact book contents.
-You should be fimilar with Spec UI framework to follow this example.
+Before we continue, we should write a simple UI which will show the contact book's contents.
+This example uses the Spec UI framework. It's documentation is SEE HERE.
```
Comment (seandenigris) [inline Chapters/Commander/Commander.pillar]: Add Spec documentation link

File: Chapters/Commander/Commander.pillar (hunk 1 above)
Comment (dionisiydk) [reply Chapters/Commander/Commander.pillar]: It is in dev branch. I will make new release soon

File: Chapters/Commander/Commander.pillar (hunk 2 above)
Comment (dionisiydk) [reply Chapters/Commander/Commander.pillar]: Ah, yes. It should be 
      ContactBook class >> family
And yes, it is class side instance var

Comment (dionisiydk) [reply Chapters/Commander/Commander.pillar]: And I fixed it

File: Chapters/Commander/Commander.pillar (hunk 3 above)
Comment (dionisiydk) [reply Chapters/Commander/Commander.pillar]: Yes, it is covered at section "Complex contexts"

**Pull request merged**